docs/getting_started.rst

```diff
@@ -205,8 +226,9 @@ Now from the fully-overlaid shell, you can run the :code:`soss` instance:
     soss src/soss-ros1/examples/hello_ros.yaml
 
 
-In this command, the executable :code:`soss` is given a :code:`.yaml` configuration file to describe how messages should be passed
-among whichever middlewares (in this case, ROS1 and ROS2).
+In this command, the executable :code:`soss` is given a :code:`.yaml` configuration file to describe how messages
```

Comment:
```suggestion
In this command, the executable :code:`soss` is given a :code:`YAML` configuration file to describe how messages
```

If you prefer this acronym, replace everywhere.

Reply:
ok.